Libs/CodeGen/ApiGen/build.gradle

```diff
@@ -75,9 +75,9 @@ project.tasks.create(name: 'genCurtisDoc',  type: JavaExec, group: taskGroup) {
             '-l',
             'CurtisDoc',
             '-o',
```

Comment:
We should probably fix this at some point. Maybe have an ENV variable that points to the path, and have it defaulting to this ../../../ stuff...